articles/article11_rag.md

```
@@ -0,0 +1,237 @@
+日本語キーワード抽出とベクトル検索の両方が機能し、AWS 関連の記事がヒットするようになっています。まだ改善の余地はありますが、一歩前進といったところです 🍵
+
+---
+
+## まとめ
+
+RAG の精度問題は「LLM が賢くない」のではなく、**LLM に渡している情報が適切でないこと**が原因であるケースが多くあります。
+
+今回の改善ポイント：
+
+| 問題 | 対策 |
+|------|------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)